### PR TITLE
formatted test framework files.

### PR DESCRIPTION
**Pull request by v-afrafi**

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlBigInt.java [hunk 1]
```diff
@@ -31,13 +14,9 @@
...
-        super("bigint", 
-				JDBCType.BIGINT, 
-				19,	//precision
-				0,	//scale
-				SqlTypeValue.BIGINT.minValue,
-				SqlTypeValue.BIGINT.maxValue,
-				SqlTypeValue.BIGINT.nullValue);
+        super("bigint", JDBCType.BIGINT, 19,	// precision
+                0,	// scale
+                SqlTypeValue.BIGINT.minValue, SqlTypeValue.BIGINT.maxValue, SqlTypeValue.BIGINT.nullValue);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlBigInt.java]: i think this change should be omitted, it doesn't look as good after formatting

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlBit.java [hunk 2]
```diff
@@ -31,14 +14,7 @@
...
     public SqlBit() {
-        super("bit", 
-				JDBCType.BIT, 
-				1,	//precision
-				0,	//scale
-				SqlTypeValue.BIT.minValue,
-				SqlTypeValue.BIT.maxValue,
-				SqlTypeValue.BIT.nullValue, 
-				VariableLengthType.Fixed);
+        super("bit", JDBCType.BIT, 1, 0, SqlTypeValue.BIT.minValue, SqlTypeValue.BIT.maxValue, SqlTypeValue.BIT.nullValue, VariableLengthType.Fixed);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlBit.java]: this one too

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlDecimal.java [hunk 3]
```diff
@@ -38,22 +21,28 @@ public SqlDecimal() {
...
-        this(name, 
-                jdbctype, 
-                38,	// precision
-                0,	// scale
-                SqlTypeValue.DECIMAL.minValue, 
-                SqlTypeValue.DECIMAL.maxValue, 
-                VariableLengthType.Scale);
+    SqlDecimal(String name,
+            JDBCType jdbctype) {
+        this(name, jdbctype, 38, 0, SqlTypeValue.DECIMAL.minValue, SqlTypeValue.DECIMAL.maxValue, VariableLengthType.Scale);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlDecimal.java]: and this

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlFloat.java [hunk 4]
```diff
@@ -31,33 +14,31 @@
...
-		super("float", 
-				JDBCType.DOUBLE, 
-				53,	//default precision
-				0,	//scale
-				SqlTypeValue.FLOAT.minValue,
-				SqlTypeValue.FLOAT.maxValue,
-				SqlTypeValue.FLOAT.nullValue, 
-				VariableLengthType.Precision);
+        super("float", JDBCType.DOUBLE, 53,	// default precision
+                0,	// scale
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlFloat.java]: here too

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlInt.java [hunk 5]
```diff
@@ -31,19 +14,12 @@
...
 
     public SqlInt() {
-        super("int", 
-				JDBCType.INTEGER, 
-				10,	//Precision
-				0,	//Scale
-				SqlTypeValue.INTEGER.minValue,
-				SqlTypeValue.INTEGER.maxValue,
-				SqlTypeValue.INTEGER.nullValue);
+        super("int", JDBCType.INTEGER, 10, 0, SqlTypeValue.INTEGER.minValue, SqlTypeValue.INTEGER.maxValue, SqlTypeValue.INTEGER.nullValue);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlInt.java]: here

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlTinyInt.java [hunk 6]
```diff
@@ -31,18 +14,13 @@
...
-		super("tinyint", 
-				JDBCType.TINYINT, 
-				3,	//precision
-				0,	//scale
-				SqlTypeValue.TINYINT.minValue,
-				SqlTypeValue.TINYINT.maxValue,
-				SqlTypeValue.TINYINT.nullValue);
-	}
+        super("tinyint", JDBCType.TINYINT, 3,	// precision
+                0,	// scale
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlTinyInt.java]: and this one

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlSmallMoney.java [hunk 7]
```diff
@@ -1,38 +1,17 @@
...
 
     public SqlSmallMoney() {
-        super("smallmoney", 
-				10, 
-				4,	// max scale for money/smallmoney
-				SqlTypeValue.SMALLMONEY.minValue,
-				SqlTypeValue.SMALLMONEY.maxValue, 
-				VariableLengthType.Fixed);
+        super("smallmoney", 10, 4,	// max scale for money/smallmoney
+                SqlTypeValue.SMALLMONEY.minValue, SqlTypeValue.SMALLMONEY.maxValue, VariableLengthType.Fixed);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlSmallMoney.java]: and this one

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlSmallInt.java [hunk 8]
```diff
@@ -31,19 +14,14 @@
...
     public SqlSmallInt() {
-        super("smallint", 
-				JDBCType.SMALLINT, 
-				5,	//Precision
-				0,	//Scale
-				SqlTypeValue.SMALLINT.minValue,
-				SqlTypeValue.SMALLINT.maxValue,
-				SqlTypeValue.SMALLINT.nullValue);
+        super("smallint", JDBCType.SMALLINT, 5,	// Precision
+                0,	// Scale
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlSmallInt.java]: here

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlReal.java [hunk 9]
```diff
@@ -30,12 +13,7 @@
...
     public SqlReal() {
-        super("real", 
-				JDBCType.REAL, 
-				24, 
-				SqlTypeValue.REAL.minValue,
-				SqlTypeValue.REAL.maxValue,
-				SqlTypeValue.REAL.nullValue, 
-				VariableLengthType.Fixed);
+        super("real", JDBCType.REAL, 24, SqlTypeValue.REAL.minValue, SqlTypeValue.REAL.maxValue, SqlTypeValue.REAL.nullValue,
+                VariableLengthType.Fixed);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlReal.java]: here

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlMoney.java [hunk 10]
```diff
@@ -1,38 +1,17 @@
...
     public SqlMoney() {
-        super("money", 
-				19, 
-				4,	// max scale for money/smallmoney
-				SqlTypeValue.MONEY.minValue,
-				SqlTypeValue.MONEY.maxValue, 
-				VariableLengthType.Fixed);
-	}
+        super("money", 19, 4,	// max scale for money/smallmoney
+                SqlTypeValue.MONEY.minValue, SqlTypeValue.MONEY.maxValue, VariableLengthType.Fixed);
```
Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlMoney.java]: here

File: src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlReal.java (hunk 9 above)
Comment (v-afrafi) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlReal.java]: do u agree then if we keep it this way?

Comment (v-ahibr) [inline src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlReal.java]: yes

**Pull request merged**